lib/creation_notifier.rb

```diff
@@ -1,15 +1,13 @@
-module Creatable
+module CreationNotifier
   def notify_after_creation # after_create
-    notify_co_authors
-    return unless !self.is_a?(Series) && self.posted?
```

Comment:
Yep, I removed the include from Series a while back. (It doesn't look like it, because it still says `include Creatable`. But that's because I renamed this module to `CreationNotifier` and renamed the `HasCreatorships` module to `Creatable`.)